src/components/Editor/EditorToolbarButton.tsx

```diff
@@ -0,0 +1,41 @@
+import React, { SVGAttributes, memo } from "react";
+import { IconButton, Icons, IconsProps } from "@storybook/components";
+import "./EditorToolbarButton.css";
+
+interface EditorToolbarButtonProps {
+  tooltip?: string;
+  text?: string;
+  icon: IconsProps["icon"];
+  color?: SVGAttributes<SVGElement>["color"];
+  disabled?: boolean;
+  smallPadding?: boolean;
+  onClick?: () => void;
+}
+
+const EditorToolbarButton: React.FC<EditorToolbarButtonProps> = ({
+  tooltip,
+  text,
+  icon,
+  color,
+  disabled,
+  smallPadding,
+  onClick,
+}) => (
+  <IconButton
+    placeholder={text}
+    title={tooltip}
+    onClick={onClick}
+    disabled={disabled}
+    className={`icon-button${smallPadding ? " small-padding" : ""}`}
+    style={{
+      margin: 0,
+      padding: `8px ${smallPadding ? "8px" : "16px"}`,
```

Comment:
Why both here and also in the className?

Reply:
will move to css (&css modules) on a future PR for it. I can remove the stylesheet files for now